.gitignore

```diff
@@ -1,8 +1,8 @@
 docs/build
 docs/site
-docs/src/*.md
 docs/src/assets/*.svg
 test/scratch.jl
 tune.json
 Manifest.toml
 settings.json
+docs/jmd
```

Comment:
Why is this?

Reply:
The idea is to eliminate the jmd directory altogether and just use an src directory now that the weaving is no longer done.